src/sql/sqlops.proposed.d.ts

```diff
@@ -435,6 +436,11 @@ declare module 'sqlops' {
 		html?: string;
 	}
 
+	export interface EditorProperties {
```

Comment:
Please add comment. I know not everything has comment here but we're trying to add comment to all